web3_proxy/tests/common/get_rpc_key.rs

```diff
@@ -0,0 +1,58 @@
+use crate::TestApp;
+use ethers::prelude::{LocalWallet, Signer};
+use ethers::types::Signature;
+use reqwest::Response;
+use rust_decimal::Decimal;
+use serde::Deserialize;
+use tracing::info;
+use web3_proxy::frontend::admin::AdminIncreaseBalancePost;
+use web3_proxy::frontend::users::authentication::{LoginPostResponse, PostLogin};
+
+#[derive(Debug, Deserialize)]
+pub struct RpcKeyResponse {
+    pub user_id: u64,
+    pub user_rpc_keys: std::collections::HashMap<String, RpcKey>,
+}
+
+#[derive(Debug, Deserialize)]
```

Comment:
can we re-use an existing type, or do we not have this exact layout already somewhere else?

Reply:
we could, i just did it for convenience right now. i'll check which ones we can re-use